calculator.js

```diff
@@ -0,0 +1,25 @@
+function
+  function addition (num1, num2){
+    return num1 + num2;
+    console.log(addtion(num1, num2));
```

Comment:
This line wouldn't run! That's because a `return` statement *exits* from the function, so code below it won't run.